run-build.ps1

```diff
@@ -110,11 +110,11 @@ $env:PATH = "$env:DOTNET_INSTALL_DIR;$env:PATH"
 if ($NoBuild)
 {
     Write-Host "Not building due to --nobuild"
-    Write-Host "Command that would be run: 'dotnet msbuild build.proj /m /p:Architecture=$Architecture $ExtraParameters'"
+    Write-Host "Command that would be run: 'dotnet msbuild build.proj /m /p:OverrideArchitecture=$Architecture $ExtraParameters'"
 }
 else
 {
-    dotnet msbuild build.proj /p:Architecture=$Architecture /p:GeneratingPropsFile=true /t:WriteDynamicPropsToStaticPropsFiles
-    dotnet msbuild build.proj /m /v:diag /p:Architecture=$Architecture $ExtraParameters
+    dotnet msbuild build.proj /p:OverrideArchitecture=$Architecture /p:GeneratingPropsFile=true /t:WriteDynamicPropsToStaticPropsFiles
```

Comment:
I wanted to rename it. So that passing something in from the command-line can override that value stored in the props file. To allow x86 build on x64 machine for example.